External/pubchem_shape/PubChemShape.cpp

```diff
@@ -272,7 +272,7 @@ ShapeInput PrepareConformer(const ROMol &mol, int confId, bool useColors) {
         throw ValueErrorException("No VdW radius for atom with Z=" +
                                   std::to_string(Z));
       }
-      rad_vector[i] = vdw_radii.at(Z);
+      rad_vector[activeAtomIdx++] = vdw_radii.at(Z);
```

Comment:
Small suggestion for this block (it's a shame Github doesn't allow you to suggest for unchanged lines). Since we already doing a look up for `vdw_radii.find(Z)`, we can avoid doing the second (index-checked!) lookup:

```
      auto zRadius = vdw_radii.find(Z);
      if (zRadius == vdw_radii.end()) {
        throw ValueErrorException("No VdW radius for atom with Z=" +
                                  std::to_string(Z));
      }
      rad_vector[activeAtomIdx++] = zRadius->second;
```